internal/pkg/service/buffer/worker/service/file_import.go

```diff
@@ -96,8 +96,22 @@ func (s *Service) importFiles(ctx context.Context, wg *sync.WaitGroup, d depende
 				}
 				tables := table.NewManager(api)
 
-				// Import file
-				if err := tables.ImportFile(ctx, fileRes); err != nil {
+				if fileRes.StorageJob == nil {
```

Comment:
Toto je ok z hladiska, ze vypadne worker a nejaky iny sa na to napoji :heavy_check_mark: 

Ale rozbije to retry, ked job spadol a ma sa to skusi neskor znova.
Teda v tomto kroku:
https://github.com/keboola/keboola-as-code/blob/041c70377d01d662da603470b895dea59ec9700f/internal/pkg/service/buffer/worker/service/file_retry.go#L39

By sa mal ten `StorageJob` nastavit na `null`. Meni sa tam state z `failed -> importing`.

Reply:
Testy chcem dnes pozriet, riesil som to aj s Honzom, kde testy nestacia na pokrytie bugu:
https://github.com/keboola/keboola-as-code/pull/1207#issuecomment-1441982487